pipelines/RobotCar/robotcar_pairs_from_retrieval.py

```diff
@@ -22,6 +21,7 @@ def main(descriptors, output, num_matched,
         lambda _, obj: h5_names.append(obj.parent.name.strip('/'))
         if isinstance(obj, h5py.Dataset) else None)
     h5_names = list(set(h5_names))
+    print(len(h5_names))
```

Comment:
I suggest to remove the prints and newlines

Reply:
Removed.